nifi-nar-bundles/nifi-ldap-iaa-providers-bundle/nifi-ldap-iaa-providers/src/main/java/org/apache/nifi/ldap/LdapProvider.java

```diff
@@ -193,8 +192,41 @@ public final void onConfigured(final LoginIdentityProviderConfigurationContext c
 
         final LdapUserSearch userSearch = new FilterBasedLdapUserSearch(userSearchBase, userSearchFilter, context);
 
-        // bind
-        final BindAuthenticator authenticator = new BindAuthenticator(context);
+
+        String rawAuthenticatorType = configurationContext.getProperty("Authenticator Type");
+        AuthenticatorType authenticatorType;
+
+        if (StringUtils.isBlank(rawAuthenticatorType))
```

Comment:
Similarly to above, braces are encouraged to be on the same line as control statements to reduce code height. You can download the checkstyle rules for your IDE and have them automatically applied.